### PR TITLE
Dynamic tweaks

### PR DESCRIPTION
**Pull request by Cyprex**
## About The Pull Request

Tweak dynamic to lower threat generated below 30 pop & add a 25% chance to just make things 0 threat.

### CODE REVIEW

**Comment by Cyprex** [comment]
uh oh, I based the branch for this PR off of the wrong branch.

File: code/game/gamemodes/dynamic/dynamic.dm [hunk 1]
```diff
@@ -152,15 +152,15 @@ GLOBAL_VAR_INIT(dynamic_forced_threat_level, -1)
...
+	var/low_pop_player_threshold = 30
 
 	/// The maximum threat that can roll with *zero* players.
 	/// As the number of players approaches `low_pop_player_threshold`, the maximum
 	/// threat level will increase.
 	/// For example, if `low_pop_maximum_threat` is 50, `low_pop_player_threshold` is 20,
 	/// and the number of readied players is 10, then the highest threat that can roll is
 	/// lerp(50, 100, 10 / 20), AKA 75.
-	var/low_pop_maximum_threat = 40
+	var/low_pop_maximum_threat = 0
```
[the remark this answers is not in the document]
Comment (Cyprex) [reply code/game/gamemodes/dynamic/dynamic.dm:163]: Maximum threat with 10 players is 33, please read the comment above.

File: code/game/gamemodes/dynamic/dynamic.dm [hunk 2]
```diff
@@ -378,6 +378,9 @@ GLOBAL_VAR_INIT(dynamic_forced_threat_level, -1)
 	if (SSticker.totalPlayersReady < low_pop_player_threshold)
 		threat_level = min(threat_level, LERP(low_pop_maximum_threat, max_threat_level, SSticker.totalPlayersReady / low_pop_player_threshold))
 
+
+	if(prob(25))
```
[the remark this answers is not in the document]
Comment (Cyprex) [reply code/game/gamemodes/dynamic/dynamic.dm:382]: > Threat increases as the round goes on.
This is news to me, what makes you say that?

**Comment by nevimer** [comment]
stale

**Pull request closed without merging**